frontend/public/components/monitoring/alerting.tsx

```diff
@@ -2321,16 +2343,22 @@ const AlertingPage: React.FC<RouteComponentProps<{ url: string }>> = ({ match })
           <Link to={rulesPath}>{t('public~Alerting rules')}</Link>
         </Tab>
       </ul>
-      <Switch>
-        <Route path={alertsPath} exact component={AlertsPage} />
-        <Route path={rulesPath} exact component={RulesPage} />
-        <Route path={silencesPath} exact component={SilencesPage} />
-      </Switch>
+      {url === alertsPath && <AlertsPage />}
```

Comment:
@kyoto could you please take a look at these file changes, if it looks good from your perspective?